src/app/App.js

```diff
@@ -32,33 +38,61 @@ class App extends React.Component {
   handleClick = e => {
     e.preventDefault();
     let newWords = this.state.words.split('').reverse().join('');
-    this.setState({  words: newWords });
+    this.setState({ words: newWords });
   }
+
   getResults = (results, count) => {
     console.log(results)
     this.setState({ results })
     this.setState({ count })
-    
+
+  }
+
+  postResults = (results, count) => {
+    console.log(`postResults function`, results)
+    this.setState({ results });
   }
+
   render() {
     return (
+
+
       <>
-        <Header 
-        headers={this.state.headers}
+        <Header
+          headers={this.state.headers}
         />
-        <Form 
-        url={"https://pokeapi.co/api/v2/pokemon"}
-        giveAppResults = {this.getResults}
-        saveHeaders = {this.saveHeaders}
-        createAppResults = {this.postResults}
-        />
-        <Results 
-        results = {this.state.results}
-        count = {this.state.count}
-        headers = {this.state.saveHeaders}
-        />
-        <Footer />
+        <BrowserRouter>
+
+          <Switch>
+            <Route exact path="/">
+              <Home />
+            </Route>
+            <Route path="/history">
+
+            </Route>
+            <Route path="/help">
+
+            </Route>
+          </Switch>
+          <Main />
+          <List>
+            <History />
+          </List>
+          <Form
+            giveAppResults={this.getResults}
+            saveHeaders={this.saveHeaders}
+            postAppResults={this.postResults}
+          />
+          <Results
+            results={this.state.results}
+            count={this.state.count}
+            headers={this.state.headers}
+          />
+
+          <Footer />
+        </BrowserRouter>
       </>
+
     )
   }
 }
```

Comment:
Stuff that I would like to see added to this:
- Help page added with information on how to use the application
- History section on the page

Your footer can be changed to the date and your name so that you get credit for all the code that you wrote for this application!